### ISSUE TITLE
Unhandled exception executing 'f claim fill player' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)

### ISSUE THREAD

**Issue by L4BORG** [MassiveCraft/Factions#1233]
## Affected Plugin

- **Affected Plugin Name:** Factions
- **Affected Plugin Version:** 2.10.0
- **MassiveCore Version:** 2.10.0

## Describe your environment

- **Spigot version:** CraftBukkit version git-TacoSpigot-"65fd35f" (MC: 1.8.8) (Implementing API version 1.8.8-R0.2-SNAPSHOT)
- **Config pastebin:** 

## Steps to reproduce this bug

```
[13:35:45 INFO]: player issued server command: /f claim fill player
[13:35:45 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f claim fill player' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:150) ~[patched.jar:git-TacoSpigot-"65fd35f"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1194) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1006) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-TacoSpigot-"65fd35f"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-TacoSpigot-"65fd35f"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-TacoSpigot-"65fd35f"]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.NullPointerException: pss
        at com.massivecraft.factions.entity.MPlayer.tryClaim(MPlayer.java:855) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsSetX.perform(CmdFactionsSetX.java:53) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:973) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:929) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:929) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:61) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:143) ~[patched.jar:git-TacoSpigot-"65fd35f"]
        ... 14 more
```

**Observed results**
Exceptions in console.

**Expected results**
Error message or something (for player, not in console).

**Comment by MarkehMe** [comment]
Please replicate in Spigot or PaperSpigot as TacoSpigot has had a lot of issues that I've seen.

**Comment by L4BORG** [comment]
```
[00:08:23 INFO]: Player issued server command: /f claim fill
[00:08:23 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f claim fill' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-PaperSpigot-"4c7641d"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-PaperSpigot-"4c7641d"]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.NullPointerException: pss
        at com.massivecraft.factions.entity.MPlayer.tryClaim(MPlayer.java:855) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsSetX.perform(CmdFactionsSetX.java:53) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:973) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:929) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:929) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:61) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) ~[patched.jar:git-PaperSpigot-"4c7641d"]
        ... 14 more
...

>ver
[00:08:35 INFO]: This server is running CraftBukkit version git-PaperSpigot-"4c7641d" (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)

```

**Comment by ulumulu1510** [comment]
Please update to the newest version of MassiveCore & Factions. 2.10 is outdated and it very well may be that this bug has been fixed already.

**Comment by L4BORG** [comment]
Can't update because new version will not load: https://github.com/MassiveCraft/Factions/issues/1221

**Comment by ulumulu1510** [comment]
#1221 and respectively #1220 have been resolved, please update to 2.13

**Comment by L4BORG** [comment]
The issue is still there on latest factions.
```
[21:32:27 INFO]: Player issued server command: /f claim fill
[21:32:27 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'f claim fill' in com.massivecraft.massivecore.command.MassiveCoreBukkitCommand(f)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:150) ~[patched.jar:git-TacoSpigot-"c2835df"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) ~[patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-TacoSpigot-"c2835df"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_121]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_121]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-TacoSpigot-"c2835df"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-TacoSpigot-"c2835df"]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.NullPointerException: pss
        at com.massivecraft.factions.entity.MPlayer.tryClaim(MPlayer.java:831) ~[?:?]
        at com.massivecraft.factions.cmd.CmdFactionsSetX.perform(CmdFactionsSetX.java:54) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1137) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1093) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCommand.execute(MassiveCommand.java:1093) ~[?:?]
        at com.massivecraft.massivecore.command.MassiveCoreBukkitCommand.execute(MassiveCoreBukkitCommand.java:60) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:143) ~[patched.jar:git-TacoSpigot-"c2835df"]
        ... 14 more
```
